.github/workflows/nvidia-test.yml

```diff
@@ -31,11 +30,25 @@ jobs:
             - name: Checkout code
               uses: actions/checkout@v4
 
+            - name: Get the artifact
+              uses: dawidd6/action-download-artifact@v6
+              with:
+                workflow: build.yml
+                name: docker_${{ github.event.inputs.run_id }}.snap
+
+            - name: Publish to Store
+              uses: snapcore/action-publish@v1
+              env:
+                SNAPCRAFT_STORE_CREDENTIALS: ${{ secrets.SNAPCRAFT_LOGIN }}
+              with:
+                snap: docker_${{ github.event.inputs.run_id }}.snap
+                release: latest/edge/runid-${{ github.event.inputs.run_id }}
+
             - name: Create Testflinger job queue
               run: |
                 export JOB_QUEUE="${{ matrix.job-queue }}"
-                export SNAP_CHANNEL="${{ inputs.snap-channel }}"
-                
+                export SNAP_CHANNEL="latest/edge/runid-${{ github.event.inputs.run_id }}"
+
```

Comment:
All instances of `github.event.inputs.run_id` can be simplified to `inputs.run_id`, as in the removed code on line 37. I see this the former mentioned in a [blog post from 2020](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/), but official docs reference the [manual trigger](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/triggering-a-workflow) inputs as latter.

Reply:
Done!

Reply:
Thanks but this isn't done. Only one instance has been changed.

Reply:
True! Fixed the other on https://github.com/canonical/docker-snap/pull/170/commits/1a51e01f4f059f88f998c3b21746fccd066916c8

Reply:
THERE ARE TWO MORE!